RUFAS/biophysical/animal/herd_manager.py

```diff
@@ -831,6 +835,21 @@ def initialize_pens(
             housing_type = pen_data.get("housing_type", "")
             pen_type = pen_data.get("pen_type", "")
             max_stocking_density = pen_data.get("max_stocking_density", 0.0)
+            minutes_away_for_milking = pen_data.get("minutes_away_for_milking", 120.0)
```

Comment:
```suggestion
            minutes_away_for_milking = pen_data.get("minutes_away_for_milking", 120)
```
The expected input type for "minutes_away_for_milking" when initializing a Pen object is `int`, setting the default to 120.0 would hint it as `float`. So either change the default here to 120 or change the expected type for input to float,

Reply:
This should probably get removed when the metadata gets updated but I will fix it for now just in case it sticks around for some reason.